protocol/blockchain/hash.md

```diff
@@ -25,8 +25,10 @@ Since [BIP-34](/protocol/forks/bip-0034), the block height is now required to be
 		 - `D5D27987D2A3DFC724E359870C6644B40E497BDC0589A033220FE15429D88599`
 		 - `E3BF3D07D4B0375638D5F1DB5255FE07BA2C4CB067CD81B84EE974B6585FB468`
 
+In contrast to many hashing algorithm implementations, Bitcoin Cash block and transaction hashes use a little-endian representation.
```

Comment:
We need to say this very precisely -- these words don't resolve all ambiguity because one might think you are referring to a little-endian double SHA-256 calculation.  How about:

As per standards, such as SHA-256, hashing algorithm implementations produce a byte array as output.  At times, the Bitcoin Cash protocol treats this byte array as a number, for example when comparing with block difficulty during block validation or mining.  In these situations, this byte array is interpreted as a 256 bit number in little-endian format.  Note that a consequence of these statements is that the hash can be imagined to be sent over the network and encoded on disk in "little-endian" format, which is the opposite of standard protocol design.  So it may be simpler to think of hashes as byte arrays that occasionally are turned into little-endian numbers, than as numbers with a lot of display/encoding caveats.